src/main/kotlin/StringAddCalculator.kt

```diff
@@ -0,0 +1,57 @@
+class StringAddCalculator {
+    fun add(text: String?): Int {
+        if (text == null || text.trim().isEmpty()) return 0
+
+        val delimiters = mutableListOf<String>()
+            .apply { addAll(DEFAULT_DELIMITERS) }
+
+        if (existCustomizedDelimiter(text)) {
+            delimiters.apply { addAll(getCustomizedDelimiter(text)) }
+            return calculate(getTrimmedText(text), delimiters)
+        }
+
+        return calculate(text, delimiters)
+    }
+
+    private fun existCustomizedDelimiter(text: String): Boolean {
+        return text.indexOf("//") == 0 && text.indexOf("\n") != -1
+    }
+
+    private fun getCustomizedDelimiter(text: String): List<String> {
+        val startIndex = text.indexOf(CUSTOM_DELIMITER_START_FLAG) + CUSTOM_DELIMITER_START_FLAG.length
+        val endIndex = text.indexOf(CUSTOM_DELIMITER_END_FLAG, startIndex)
+        return text.substring(startIndex, endIndex)
+            .map{ it.toString() }
+            .onEach { char ->
+                runCatching {
+                    require(!char.equals(MINUS_OPERATOR)) { }
```

Comment:
코틀린은 `equals` 비교를 `==`으로 제공하고 있어요. [참고](https://kotlinlang.org/docs/operator-overloading.html#equality-and-inequality-operators)하시면 좋을 것 같습니다.
```suggestion
                    require(char != MINUS_OPERATOR))
```